CZ/pyranda_rayleigh_taylor/rayleigh_taylor.py

```diff
@@ -248,21 +251,28 @@ def getPressure(pysim,p0):
     p0     = 1.0
     At     = ( (rho_h - rho_l)/(rho_h + rho_l) )
     # NOTE: WHY THIS PARTICULAR FRACTION OF THE RT GROWTH RATE?
-    u0     = sqrt( abs(gx*At/waveLength) ) * .1
+
 
     # Add interface
     # NOTE: diffuse interface?
-    :Yl: = .5 * (1.0-tanh( sqrt(pi)*( meshx - 1.4*pi ) / delta ))
+    :Yl: = .5 * (1.0-tanh( sqrt(pi)*( meshx ) / delta ))
     :Yh: = 1.0 - :Yl:
     :p:  += p0 
 
     # Add oscillations near interface
     wgt = 4*:Yh:*:Yl:
     :v: *= 0.0
     :w: *= 0.0
-    # NOTE: what is gbar?
+
+    # gbar is 2D gaussian filter (smoothes stuff out)
+    # Add random velocity (filtered white noise)
+    u0  = sqrt( abs(gx*At/waveLength) ) * ranVelMag
     :u: = wgt * gbar( (random3D()-0.5)*u0 )
 
+    # Add regular modes
+    u1  = sqrt( abs(gx*At/waveLength) ) * velMag
+    :u: = :u: + wgt * u1 * - cos( meshy * velMod )
```

Comment:
Can this actually add more than one mode, or is it just adding a single prescribed mode dependent upon the domain height?

Reply:
velMod = 4 would add 4 modes, yeah.